modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -12,7 +12,7 @@ template <>
 InputParameters
 validParams<RadialReturnStressUpdate>()
 {
-  InputParameters params = validParams<StressUpdateBase>();
+  InputParameters params = validParams<StateUpdateBase>();
   params.addClassDescription("Calculates the effective inelastic strain increment required to "
                              "return the isotropic stress state to a J2 yield surface.  This class "
                              "is intended to be a parent class for classes with specific "
```

Comment:
**Not done**.  The problem here is this hair-pulling base_name stuff.  Now RadialReturnStressUpdate doesn't know anything about the elasticity tensor except for through updateState.  It would be too expensive to check the elasticity tensor is isotropic every time updateState is made.  We could add another parameter to updateState (bool elasticity_tensor_is_isotropic) if you like.